### PR TITLE
feat(content): add page for wolle

### PR DESCRIPTION
**Pull request by Tirokk**
## 🍰 Pullrequest

Add Page for Wolle

### Issues

- None

### Todo

- [X] None

### CODE REVIEW

File: docs/people/wolfgang-huss.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+So my projects so far have been almost all graphic related, up to my high-end large format printing software `Pjannto RIP`, with color calibration for highest demands of my customers for printing on various printer brands and devices.  
+Also my iPhone App [FaceYourFace](https://apps.apple.com/de/app/faceyourface/id1137963659) for mirroring face halves to create double-left and double-right face images is a graphic application.
+
+<!-- Schließlich habe ich mich derzeit von der Entwicklung von Anwendungssoftware auf Desktop-Computern mehr in Richtung von Web-Anwendungen für Internet-Browser orientiert (Frontend).
+Dies schließt die Entwicklung von Server-Programmen (Backend) und den Einsatz von Datenbanken sowie auch die Server-Administration mit ein und kann ebenso das Projektmanagement und die Entwicklungsleitung umfassen, wie beim [ocelot.social Projekt](https://github.com/Ocelot-Social-Community). -->
+Finally, I have currently moved away from application software development for desktop computers more towards web applications for internet browsers (frontend).
```
Comment (ulfgebhardt) [inline docs/people/wolfgang-huss.md:25]:
```suggestion
Finally I have moved away from application software development for desktop computers more towards web applications for internet browsers (frontend).
```

```suggestion
Currently I have moved away from application software development for desktop computers more towards web applications for internet browsers (frontend).
```

File: docs/people/wolfgang-huss.md [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+This includes the development of server applications (backend) and the use of databases as well as server administration and can also include project management and development management, as with the [ocelot.social project](https://github.com/Ocelot-Social-Community).
+
+<!-- Einer der Gründe dafür war, gerne im Team zu arbeiten, und mit agilen Methoden Software zu entwickeln. -->
+One of the reasons was to enjoy working in a team, and to develop software using agile methods.
+
+<!-- Und nicht zu letzt war ich 12 Jahre Inhaber einer „kleinen“ Werbeagentur (8 Mitarbeiter) in der wir Logos, Prospekte, Kataloge erstellt und als Schauwerbegestalter für die Anfertigung von Schildern, Ladenbeschriftungen, Messetafeln und vielem anderen tätig waren.
+Dabei kamen die gängigen Grafikprogramme zum Einsatz.  
+Dies kommt mir heute auch bei der programmierung zu gute. -->
+And not at least, for 12 years I was the owner of a "small" advertising agency (8 employees) in which we created logos, designs, brochures, catalogs etc.
```
Comment (ulfgebhardt) [inline docs/people/wolfgang-huss.md:34]:
```suggestion
And last but not least, for 12 years I was the owner of a "small" advertising agency (8 employees) in which we created logos, designs, brochures, catalogs etc.
```

**Review by ulfgebhardt — CHANGES_REQUESTED** [review]

File: docs/people/wolfgang-huss.md (hunk 2 above)
Comment (Tirokk) [reply docs/people/wolfgang-huss.md:34]: This one of the things the machine critizised 🤣

I like to have it in again.

File: docs/people/wolfgang-huss.md (hunk 1 above)
Comment (Tirokk) [reply docs/people/wolfgang-huss.md:25]: cool

**Review by ulfgebhardt — APPROVED** [review]

**Pull request merged**